### PR TITLE
add more details about helm support to kustomize docs

### PR DESCRIPTION
**Pull request by natasha41575**
/cc @KnVerey

### CODE REVIEW

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Charts"
+linkTitle: "Helm Charts"
+type: docs
+weight: 6
+description: >
+Helm chart inflation generator
```
Comment (KnVerey) [inline site/content/en/references/kustomize/kustomization/helmcharts/_index.md:7]: indentation is missing here, which I believe is preventing the page from being built (it's not in the preview site)

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The helm chart inflation generator in kustomize is intended to be a limited subset of helm features to help with
+getting started with kustomize, and we cannot support the entire helm feature set.
+
+### The current builtin
+For enhancements to the helm chart inflation generator feature, we will only support the following changes:
+
+- bug fixes
+- critical security issues
+- additional fields that are analogous to flags passed to `helm template`
```
Comment (KnVerey) [inline site/content/en/references/kustomize/kustomization/helmcharts/_index.md:28]: with the exception of flags that enable arbitrary command execution

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+We will not add support for:
+- private repository or registry authentication
+- OCI registries
+- other large features that increase the complexity and security of the feature
```
Comment (KnVerey) [inline site/content/en/references/kustomize/kustomization/helmcharts/_index.md:33]:
```suggestion
- other large features that increase the complexity of the feature and/or have significant security implications
```

(as you said above, we will fix security issues)

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+type: docs
+weight: 6
+description: >
+Helm chart inflation generator
+---
+
+[kustomize builtins]: https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_helmchartinflationgenerator_
+[Helm support long term plan]: https://github.com/kubernetes-sigs/kustomize/issues/4401
+
+## Helm Chart Inflation Generator
```
Comment (KnVerey) [inline site/content/en/references/kustomize/kustomization/helmcharts/_index.md:13]: This page and/or the builtins reference should probably mention the flags required for this to work

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md (hunk 2 above)
Comment (natasha41575) [reply site/content/en/references/kustomize/kustomization/helmcharts/_index.md]: done

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md (hunk 3 above)
Comment (natasha41575) [reply site/content/en/references/kustomize/kustomization/helmcharts/_index.md]: done

File: site/content/en/references/kustomize/kustomization/helmcharts/_index.md (hunk 1 above)
Comment (natasha41575) [reply site/content/en/references/kustomize/kustomization/helmcharts/_index.md]: Thank you for your help! This is fixed now

**Pull request merged**